definitions/npm/react-redux_v5.x.x/flow_v0.62.0-/react-redux_v5.x.x.js

```diff
@@ -0,0 +1,95 @@
+import type { Dispatch, Store } from "redux";
```

Comment:
I don't this this works, I think `Dispatch` and `Store` just fallback to `any`.

Reply:
That's right. It's just copy paste from previous libdefs. When moving the import inside module declaration, they start to work, but tests start to fail as flow-typed cannot find the redux module.